spec/models/like_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.create(name: 'George', photo: 'www.sweetimage.com', bio: 'Super smart')
+  post = Post.create(user:, title: 'Title', text: 'sweet text!')
+
+  subject { Like.new(user:, post:) }
+
+  before { subject.save }
+end
```

Comment:
- [ ] You are almost there! However, there's still missing one test `CommentsCounter must be an integer greater than or equal to zero.`, as the last reviewer said, this test was not passing but it was deleted:

<img width="592" alt="Screen Shot 2022-07-23 at 10 44 16 PM" src="https://user-images.githubusercontent.com/83717285/180631286-114ffae8-84d2-425e-b3b3-4d75bc8cfa14.png">
 
Please add a Likes test as required by the activity.